src/lib/seam/components/AccessCodeTable/AccessCodeTable.tsx

```diff
@@ -296,4 +330,5 @@ const t = {
   accessCodes: 'Access Codes',
   noAccessCodesMessage: 'Sorry, no access codes were found',
   loading: 'Loading access codes',
+  codeWasSaved: 'Code was saved',
```

Comment:
```suggestion
  accessCodeSaved: 'Access code saved',
```

Reply:
Is it possible to have a special message `Access code created` for the create case?

```suggestion
  accessCodeUpdated: 'Access code updated',
  accessCodeCreated: 'Access code created',
```

Reply:
Could do, updated in https://github.com/seamapi/react/pull/517/commits/0bc35b0f17ab93ca0db82559bfac1998a274ce7d